Mathlib/RingTheory/RingHom/FiniteType.lean

```diff
@@ -100,4 +101,14 @@ theorem finiteType_respectsIso : RingHom.RespectsIso @RingHom.FiniteType :=
   RingHom.finiteType_is_local.respectsIso
 #align ring_hom.finite_type_respects_iso RingHom.finiteType_respectsIso
 
+theorem finiteType_stableUnderBaseChange : StableUnderBaseChange @FiniteType := by
```

Comment:
Why aren't you adding a `StableUnderBaseChange @FinitePresentation` theorem?

Reply:
Because the file `RingHom/FinitePresentation.lean` does not yet exist. I am happy to follow-up on that though with another PR.